apps/examples/st_things/light/sample_light_user.c

```diff
@@ -57,23 +58,34 @@ bool handle_get_request_on_switch(st_things_get_request_message_s *req_msg, st_t
 bool handle_set_request_on_switch(st_things_set_request_message_s *req_msg, st_things_representation_s *resp_rep)
 {
 	printf("[%s]IN-handle_set_request_on_switch() called..\n", TAG);
-
-	char *power;
-	if (req_msg->rep->get_str_value(req_msg->rep, PROPERTY_VALUE_SWITCH, &power)) {
-		printf("[%s] power : %s\n", TAG, power);
-		if (strncmp(power, "off", sizeof("off")) == 0) {
+	if (req_msg->rep->get_str_value(req_msg->rep, PROPERTY_VALUE_SWITCH, &g_power)) {
+		printf("[%s] power : %s\n", TAG, g_power);
+		if (strncmp(g_power, "off", sizeof("off")) == 0) {
 			g_switch_value = 0;
 		} else {
 			g_switch_value = 1;
 		}
-		resp_rep->set_str_value(resp_rep, PROPERTY_VALUE_SWITCH, power_status[g_switch_value]);
+		resp_rep->set_str_value(resp_rep, PROPERTY_VALUE_SWITCH, g_power_status[g_switch_value]);
 		st_things_notify_observers(req_msg->resource_uri);
 	}
 
 	printf("[%s]OUT-handle_set_request_on_switch() called..\n", TAG);
 	return true;
 }
 
+//change value of switch using switch(702) at artik board
```

Comment:
1.  Add a space after // and first letter of the comment should be upper case.
2.  Change the function name to make it more meaningful. Ex: change_switch_value.

Reply:
fixed.